src/strategies/degenzoo-erc721-animals-weighted/README.md

```diff
@@ -0,0 +1,8 @@
+# ERC721 with Metadata attribute
+
+This strategy allows you to determine the voting power by reading the metadata attribute of holding NFT.
+`tokenURI(tokenID)` returns the individual metadata URI in ERC721.
```

Comment:
We don't have any tokenURI in examples, can update documentation

Reply:
Updated README with information related to the way we gather Metadata